index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Video App</title>
+  <link rel="stylesheet" href="./index.css" />
+</head>
+
+<body>
+
+  <div class="main">
+
+    <nav class="navbar">
+      <div class="brand">
+        <img src="./img/menu.png" alt="menu-icon" />
+        <span>Video App</span>
+      </div>
+
+      <div class="input-field">
+        <input class="searchBar" type="text" />
+        <img src="./img/search.png" alt="search-icon" />
+      </div>
+
+      <div class="menu-section">
+        <ul class="top-menu">
+
+          <li><a href=""><img src="./img/notification.png" alt="" /></a></li>
+          <li><a href=""><img src="./img/video-camera.png" alt="" /></a></li>
+          <li><a href=""><img class="avatar" src="https://source.unsplash.com/random" alt="" /></a>
+          </li>
+
+        </ul>
+
+      </div>
+    </nav>
+
+    <section class="main-section">
+
+      <div class="r-side">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/oGYv3sCuxVQ" title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen></iframe>
+
+        <div class="title">
+          <h3>What does Earth sound like from space?</h3>
+        </div>
+
+          <ul class="icon-bar">
+
+            <li class="icon-cols">Like</li>
+            <li class="icon-cols">Dislike</li>
+            <li class="icon-cols">Save</li>
+            <li class="icon-cols">Share</li>
+            <li class="icon-cols"> More</li>
+
+          </ul>   
```

Comment:
I think it will be better to use the list tag instead of span tags there to conveniently structure your code and make styling easier 😃 👍

RE: There is no need for this comment since the changes have been made already 😃 👍